KPI/F.adoc

```diff
@@ -1,60 +1,69 @@
-  /gmd:MD_Metadata/gmd:distributionInfo/gmd:MD_Distribution/gmd:transferOptions/gmd:MD_DigitalTransferOptions/gmd:onLine/gmd:CI_OnlineResource/gmd:linkage
-  
-==== Example
-DWD ICON EPS model data, metadata record
-
-- https://gisc.dwd.de/wisportal/#SearchPlace:q?pid=de.dwd.icon-eps.ALL 
+=== WMCP element(s)
 
-Digital Transfer Options
-....
-<gmd:MD_DigitalTransferOptions>
-          <gmd:onLine>
-            <gmd:CI_OnlineResource>
-              <gmd:linkage>
-                <gmd:URL>https://opendata.dwd.de/weather/wmc/icon-eps/data/grib</gmd:URL>
-              </gmd:linkage>
-              <gmd:protocol>
-                <gco:CharacterString>http</gco:CharacterString>
-              </gmd:protocol>
-              <gmd:name>
-                <gco:CharacterString>GISC Offenbach, Deutscher Wetterdienst</gco:CharacterString>
-              </gmd:name>
-              <gmd:description>
-                <gco:CharacterString>WMO Information System, download products/data through GISC Offenbach, Deutscher Wetterdienst</gco:CharacterString>
-              </gmd:description>
-            </gmd:CI_OnlineResource>
-          </gmd:onLine>
-</gmd:MD_DigitalTransferOptions>
-....
+* `/gmd:MD_Metadata/gmd:identificationInfo//gmd:resourceConstraints/gmd:MD_LegalConstaints/gmd:otherConstraints`
+* `/gmd:MD_Metadata/gmd:distributionInfo/gmd:MD_Distribution/gmd:transferOptions/gmd:MD_DigitalTransferOptions/gmd:onLine/gmd:CI_OnlineResource/gmd:linkage`
 
+=== What is being measured
 
-=== What is being measured?
-A measure shoud be if a link to the Data is given in the Metadtata.
+Whether the DataLicenseCode of `WMOEssential` is provided as a constraint as
+part of data identification, if yes then ensure that distribution URLs are
+provided.
 
+=== Rationale for measurement
 
-=== Why should we measure it?
-All WMOEssential data should have a link. Can it be an explicit link
-(without login). Not all the time because some datasets are freely
-available but require login.
+All `WMOEssential` data should have 1..n distribution links to the data.
 
+=== Rules for implementation
 
-=== How is the measurement executed?
-Test of the DigitalTranferOptions are given.
+```xml
+<gmd:resourceConstraints>
+  <gmd:MD_LegalConstraints>
+    <gmd:otherConstraints>
+      <gmx:Anchor xlink:href="http://wis.wmo.int/2013/codelists/WMOCodeLists.xml#WMO_DataLicenseCode_WMOEssential">WMOEssential</gmx:Anchor>
+    </gmd:otherConstraints>
+  </gmd:MD_LegalConstraints>
+</gmd:resourceConstraints>
+```
+  
+```xml
+<gmd:MD_DigitalTransferOptions>
+  <gmd:onLine>
+    <gmd:CI_OnlineResource>
+      <gmd:linkage>
+        <gmd:URL>https://opendata.dwd.de/weather/wmc/icon-eps/data/grib</gmd:URL>
+      </gmd:linkage>
+      <gmd:protocol>
+        <gco:CharacterString>http</gco:CharacterString>
+      </gmd:protocol>
+      <gmd:name>
+        <gco:CharacterString>GISC Offenbach, Deutscher Wetterdienst</gco:CharacterString>
+      </gmd:name>
```

Comment:
Comment from my GISC Offenbach colleagues:
- name linked to fixed value for data link, e.g. "dataLink"